### ISSUE TITLE
Porting to Chrome,please

### ISSUE THREAD

**Issue by sugar778** [CanisLupus/swift-selection-search#184]
Hi,
I've found this extension to be a great timesaver when searching on Firefox. but I'd love to see if it could be made available on Chrome as well.

I'd really appreciate it if you would consider doing this so that more users will be able to use this excellent extension.

**Comment by CanisLupus** [comment]
Hi! First of all, let me say that I appreciate the intention and also thank you for your interest in having SSS on Chrome. :)

I have been asked this before, but this is something that is _very unlikely_ to happen. I have considered it in the past, but there are 2 things that were and still are a problem for me. :)

1. Although addons in both Firefox and Chrome use WebExtensions, some WebExtensions APIs are only available on some browsers. I'm not sure if I am using any of those in SSS, though. It's also possible that the manifest file for the extension must be different for each version, since it cannot contain browser-specific differences all in one (for example, at one point I tried porting SSS to Firefox for Android, which would not support some manifest entries from normal Firefox and so required a completely different file). This is inconvenient but not a big issue.

2. Time. This is the vastly larger reason. I don't use Chrome, so my ability to provide support and test the addon is greatly reduced, already discarding the time needed to go around any eventual browser differences in the implementation. It happens more frequently that I'd like that I make an update and break something in SSS for an older Firefox version that I still want to support. I can only imagine what would happen if I tried supporting SSS for a browser and addons market that I don't use. ;) There's also a lot more users on Chrome, which would be great if SSS was a paid addon, but the number of feature requests and questions I get _now_ scaled to the number of users on Chrome would simply be overwhelming for my free time!

So, in summary, for the foreseeable future I would really like to keep SSS at a scale where it is manageable for me, and Chrome would unfortunately break that. :p I hope that's understandable!

Cheers!
Daniel

PS: a user has previously shown me an alternative for Chrome users, which is [this addon](https://chrome.google.com/webstore/detail/simple-select-%20-search/aagminaekdpcfimcbhknlgjmpnnnmooo) (it even has the same "SSS" acronym!). I have never used it, but maybe it can help!

**Comment by CanisLupus** [comment]
Hey @sugar778, I hope you are doing well.

I'll be closing this issue since it's just not in my plans for now. I hope that's okay! Sorry, I know that this isn't a great outcome.... I appreciate your interest in this, regardless.
